src/main/java/edu/harvard/iq/dataverse/api/Datasets.java

```diff
@@ -10,6 +10,7 @@
 import edu.harvard.iq.dataverse.authorization.RoleAssignee;
```

Comment:
I'm just putting this here the top but this PR needs:

- docs
- a release note